### PR TITLE
HOUS-155: Creates generate unique tax ids method

### PR DESCRIPTION
**Pull request by DanaNussair**
## Description
Adds the `generateUniqueSteuerIds` to `validate-steuerid` package

Addresses [HOUS-155](https://kontist.atlassian.net/browse/HOUS-155)

### CODE REVIEW

**Review by UgurGumushan — APPROVED** [review]

File: test/test.js [hunk 1]
```diff
@@ -46,3 +46,19 @@ describe('isOccurrencesValid function returns false on obviously false numbers',
...
+    assert.strictEqual(taxIdArray.length, 2)
+  })
+  it('Generates valid tax ids', () => {
+    const taxIdArray = generateUniqueSteuerIds(2)
+    taxIdArray.forEach((id) => assert.strictEqual(isSteuerIdValid(id), true))
+  })
+  it('Generates a unique set of tax ids', () => {
+    const taxIdArray = generateUniqueSteuerIds(10)
+    const taxIdSet = new Set(taxIdArray)
+    assert.strictEqual(taxIdArray.length, taxIdSet.size)
```
Comment (mkhstar) [inline test/test.js:62]:
```suggestion
    const length = 10;
    const taxIdArray = generateUniqueSteuerIds(length)
    assert.strictEqual(taxIdArray.length, length)
```

File: test/test.js [hunk 2]
```diff
@@ -46,3 +46,19 @@ describe('isOccurrencesValid function returns false on obviously false numbers',
...
+describe('Generate unique tax ids', () => {
+  it('Generates the number of tax ids provided', () => {
+    const taxIdArray = generateUniqueSteuerIds(2)
+    assert.strictEqual(taxIdArray.length, 2)
+  })
+  it('Generates valid tax ids', () => {
+    const taxIdArray = generateUniqueSteuerIds(2)
+    taxIdArray.forEach((id) => assert.strictEqual(isSteuerIdValid(id), true))
+  })
+  it('Generates a unique set of tax ids', () => {
```
Comment (mkhstar) [inline test/test.js:59]: This test case is similar to the first test case in this describe block

File: README.md [hunk 3]
```diff
@@ -26,21 +26,30 @@ isSteuerIdValid('26954371827')
...
+* ### Takes `steuerId`:
   - type: `string`
-### Returns `boolean`
+* ### Returns `boolean`
 
 ## generateSteuerId()
+* ### Returns `string`
 
-### Returns `string`
+## generateUniqueSteuerIds(numberOfSteuerIds)
```
Comment (mkhstar) [inline README.md:43]: Takes for this is missing 😛

File: test/test.js (hunk 1 above)
Comment (DanaNussair) [reply test/test.js:62]: this is meant evaluate the uniqueness of the values rather than just the size of the array.

Comment (mkhstar) [reply test/test.js:62]: Thanks for clarifying

**Review by mkhstar — APPROVED** [review]

**Pull request merged**